Automattic/jetpack
# Jetpack Backup: verify that Assets package exists before calling

Fixes #
Related to [this issue.](https://github.com/Automattic/jetpack/issues/22277)
#### Changes proposed in this Pull Request:
Check that the methodAssets::alias_textdomains_from_file
method exists before calling it to avoid fatal error
whenever older version of Assets package is used.



#### Jetpack product discussion
p1641832336007500-slack-C025MP1Q3HT

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
1- Open Jurassic Ninja to launch new site
2- Click on Want more options
3- Include Jetpack and Jetpack Beta. 
4- Launch the site
5- Go to your site -> Jetpack -> Jetpack Beta -> Jetpack
6- Choose the Jetpack Version 10.4
7- Activate Jetpack Backup plugin choosing `update/add_check_for_alias_textdomains` on feature branch.
8- confirm that the site does not crash
